### PR TITLE
EVG-7427: integrate multi user manager

### PR DESCRIPTION
**Pull request by Kimchelly**
JIRA: https://jira.mongodb.org/browse/EVG-7427

* Add multi auth user manager from gimlet.
* Admin settings for multi auth.
* Fix a bug in the error notification flash where it displays an undefined field in the error response rathar then the actual error message.

### CODE REVIEW

File: rest/model/admin.go [hunk 1]
```diff
@@ -412,12 +413,6 @@ func (a *APIAuthConfig) BuildFromService(h interface{}) error {
 				return errors.Wrap(err, "could not build API Okta auth settings from service")
 			}
 		}
-		if v.Okta != nil {
```
Comment (bsamek) [inline rest/model/admin.go]: Why did you delete this?

Comment (Kimchelly) [reply rest/model/admin.go]: This is a copy-paste of the lines above this (but GitHub hides too much of the surrounding context to see it from the file preview).
